src/OceanSimulations/OceanSimulations.jl

```diff
@@ -9,59 +16,92 @@ using SeawaterPolynomials.TEOS10: TEOS10EquationOfState
+                          drag_coefficient = 0.003,
+                          momentum_advection = default_momentum_advection(),
+                          tracer_advection = default_tracer_advection(),
+                          verbose = false)
 
     # Set up boundary conditions using Field
     top_zonal_momentum_flux      = Jᵘ = Field{Face, Center, Nothing}(grid)
     top_meridional_momentum_flux = Jᵛ = Field{Center, Face, Nothing}(grid)
     top_ocean_heat_flux          = Jᵀ = Field{Center, Center, Nothing}(grid)
     top_salt_flux                = Jˢ = Field{Center, Center, Nothing}(grid)
 
-    ocean_boundary_conditions = (u = FieldBoundaryConditions(top=FluxBoundaryCondition(Jᵘ)),
-                                 v = FieldBoundaryConditions(top=FluxBoundaryCondition(Jᵛ)),
-                                 T = FieldBoundaryConditions(top=FluxBoundaryCondition(Jᵀ)),
-                                 S = FieldBoundaryConditions(top=FluxBoundaryCondition(Jˢ)))
+    u_bot_bc = FluxBoundaryCondition(u_quadratic_bottom_drag, discrete_form=true, parameters=drag_coefficient)
+    v_bot_bc = FluxBoundaryCondition(v_quadratic_bottom_drag, discrete_form=true, parameters=drag_coefficient)
+
+    ocean_boundary_conditions = (u = FieldBoundaryConditions(top = FluxBoundaryCondition(Jᵘ), bottom = u_bot_bc),
+                                 v = FieldBoundaryConditions(top = FluxBoundaryCondition(Jᵛ), bottom = v_bot_bc),
+                                 T = FieldBoundaryConditions(top = FluxBoundaryCondition(Jᵀ)),
+                                 S = FieldBoundaryConditions(top = FluxBoundaryCondition(Jˢ)))
+
+    Fu = Forcing(u_immersed_bottom_drag, discrete_form=true, parameters=drag_coefficient)
+    Fv = Forcing(v_immersed_bottom_drag, discrete_form=true, parameters=drag_coefficient)
+
+    forcing = (; u = Fu, v = Fv)
 
     # Use the TEOS10 equation of state
     teos10 = TEOS10EquationOfState(; reference_density)
     buoyancy = SeawaterBuoyancy(; gravitational_acceleration, equation_of_state=teos10)
 
     # Minor simplifications for single column grids
-    Nx, Ny, Nz = size(grid)
+    Nx, Ny, _ = size(grid)
     if Nx == Ny == 1 # single column grid
         tracer_advection = nothing
         momentum_advection = nothing
-    else
-        # TODO: better advection scheme
-        tracer_advection = WENO()
-        momentum_advection = VectorInvariant(vorticity_scheme = WENO(),
-                                             divergence_scheme = WENO(),
-                                             vertical_scheme = WENO())
     end
 
     tracers = (:T, :S)
     if closure isa CATKEVerticalDiffusivity
         tracers = tuple(tracers..., :e)
+        tracer_advection = (; T = tracer_advection, S = tracer_advection, e = nothing)
     end
 
-    coriolis = HydrostaticSphericalCoriolis(; rotation_rate)
+    coriolis = HydrostaticSphericalCoriolis(; rotation_rate, scheme = ActiveCellEnstrophyConserving())
```

Comment:
This is default

https://github.com/CliMA/Oceananigans.jl/blob/6730e6f6b2c8f1695e20b95ef467b5b14fdc4c5f/src/Coriolis/hydrostatic_spherical_coriolis.jl#L41
```suggestion
    coriolis = HydrostaticSphericalCoriolis(; rotation_rate)
```